doc/source/Resources/User_Guide.rst

```diff
@@ -1,17 +1,12 @@
-User Guide
-----------
```

Comment:
Remove the following section but keep the "User Guide" section since we need this to get the top level titles to display "User Guide"